README.md

```diff
@@ -27,36 +26,65 @@ With "read the manual" fatigue and lengthy selection processes wearing on us eve
 
-    - persistence-api: Interface for working with persistence services
-    - persistence-common: Utilities shared by the persistence services
-    - persistence-cassandra-3.11: Joins C* 3.11 cluster as coordinator-only node (does not store data),
-    mocks C* system tables for native driver integration,
-    executes requests with C* storage nodes using C* QueryHandler/QueryProcessor,
-    converts internal C* objects and ResultSets to Stargate Datastore objects.
-    - persistence-cassandra-4.0: (same as above but for Cassandra 4.0)
+- [core](core): Common classes used throughout the other coordinator modules
+- [cql](cql): API implementation for the Cassandra Query Language
+- [grpc](grpc): fast CQL over gRPC implementation (HTTP-based interface equivalent to CQL performance)
+- [bridge](bridge): gRPC-based interface used by API services
+- [health-checker](core): HTTP endpoints useful for health checking coordinator nodes
+- [metrics-jersey](core): metrics collection for the coordinator node and its exposed interfaces
+- [stargate-starter](stargate-starter): the main Java application used to start the coordinator via the `starctl` script
 
-- **Authentication Services**: Responsible for authentication to Stargate
+#### Persistence Services
+Stargate coordinator nodes support a pluggable approach for implementing the coordination layer to execute requests passed by API services and other interfaces to underlying data storage instances. Persistence service implementations are responsible handling and converting requests to db queries, dispatching to persistence, returning and serving response
 
-    - auth-api: REST service for generating auth tokens
-    - auth-table-based-service: Service to store tokens in the database
-    - authentication: Interface for working with auth providers
+- [persistence-api](persistence-api): Interface for working with persistence services.  as   
+- [persistence-common](persistence-common): Utilities shared by the persistence services
+- [persistence-cassandra-3.11](persistence-cassandra-3.11): Joins C* 3.11 cluster as coordinator-only node (does not store data),
+mocks C* system tables for native driver integration,
+executes requests with C* storage nodes using C* QueryHandler/QueryProcessor,
+converts internal C* objects and ResultSets to Stargate Datastore objects.
+- [persistence-cassandra-4.0](persistence-cassandra-4.0): (same as above but for Cassandra 4.0)
+- [persistence-dse-6.8](persistence-dse-6.8): (same as above but for DataStax Enterprise 6.8)
 
-![image](assets/stargate-modules-preview-version.png#center)
+#### Authentication Services
+Stargate coordinator nodes also support a pluggable authentication approach.
+
+- [authnz](authnz): Interface for working with auth providers
+- [auth-api](auth-api): REST service for generating auth tokens
```

Comment:
You used periods at the end of the descriptions in the last table. To be consistent, you should add them here. Or take them away in all cases. Consistency is the important point.

Reply:
will make consistent